clean-abap/CleanABAP.md

```diff
@@ -459,6 +460,23 @@ for example prefer `detection_object_types` to something cryptic like `dobjt`.
 
 > Read more in _Chapter 2: Meaningful Names: Use Pronounceable Names_ of [Robert C. Martin's _Clean Code_]
 
+### Use snake_case
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Names](#names) > [This section](#use-snake_case)
+
+ABAP is case insensitive and limits the number of available letters for naming all kind of DDIC-objects
+and variables. For exampe the maximum length of a method name is 30 characters. The defacto naming convention
+in ABAP is `snake_case`, because camelCase is overwritten when the Pretty Printer is activated and not all team
+members use the 'Keep Camel Case Identifiers' setting in the ADT ABAP Formatter. Also `camelCase` doesn't
```

Comment:
I think we should reformulate the reason given. 

The main reason for `snake_case` to be the main convention is the fact that ABAP is case insensitive and the system will disregard the case of the name in several places. This should be presented as the main argument. The fact about the pretty printer is a nice addition, but shouldn't be presented as the main argument in my opinion.

Reply:
good one.